tools/meteor-services/deploy.js

```diff
@@ -442,8 +644,20 @@ export function bundleAndDeploy(options) {
     return 1;
   }
 
+  // After an upload succeeds, we want to poll Galaxy to see if the build / deploy succeeds
+  // We used to indicate that the upload was finished via a message to check Galaxy
   if (result.payload.message) {
-    Console.info(result.payload.message);
+    // If there is a newVersionId, then poll for status of the app
+    let newVersionId = result.payload.newVersionId;
+    if(!!newVersionId) {
+      return executePollForDeploymentSuccess(newVersionId, 
+        options.deployPollingTimeoutMillis,
+      result);
```

Comment:
indentation is weird here. (maybe Meteor can switch to prettier or something? :) we're hoping to do that soon in galaxy-server)